gcp_variant_transforms/pipeline_common.py

```diff
@@ -329,3 +330,26 @@ def generate_unique_name(job_name):
   return '-'.join([job_name,
                    datetime.now().strftime('%Y%m%d-%H%M%S'),
                    str(uuid.uuid4())])
+
+
+
+BQ_CREATE_PARTITIONED_TABLE = (
+    'bq mk --table '
+    '--range_partitioning=start_position,0,{TOTAL_BASE_PAIRS},{PARTITION_SIZE} '
+    '--clustering_fields=start_position,end_position '
+    '{FULL_TABLE_ID} {SCHEMA_JSON}')
+
+def create_output_table_if_needed(known_args, full_table_id, total_base_pairs):
```

Comment:
I think it's clearer if we pass append and schema_json directly. WDYT?

Reply:
Agree, modified.